src/hooks/useQuizEvents/index.ts

```diff
@@ -28,14 +28,7 @@ const useQuizEvents: UseQuizEvents = (quizOptions, cioClient, quizState) => {
     quizSessionStorageState,
   } = quizState;
   const { callbacks } = quizOptions;
-  const {
-    onAddToCartClick,
-    onAddToFavoritesClick,
-    onQuizNextQuestion,
-    onQuizResultClick,
-    onQuizResultsLoaded,
-    onQuizSkipQuestion,
-  } = callbacks || {};
+  const { onQuizNextQuestion, onQuizSkipQuestion } = callbacks || {};
```

Comment:
Why are we changing this? Isn't it easier to de-structure everything here? (aka. the implementation on main)